test/unit/krane/container_overrides_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+module Krane
+  class ContainerOverridesTest < Krane::TestCase
+    def setup
+      super
+      @container = Kubeclient::Resource.new(
+        name: "task-runner",
+        image: "gcc:3.2",
```

Comment:
We're using `busybox` everywhere else, why `gcc` here?

Reply:
Nothing in particular. I can change if you want 😄